lib/parser/index.js

```diff
@@ -28,7 +28,8 @@ var $ = HTML.TAG_NAMES,
  */
 var DEFAULT_OPTIONS = {
     locationInfo: false,
-    treeAdapter: defaultTreeAdapter
+    treeAdapter: defaultTreeAdapter,
+    onToken: function () {}
```

Comment:
Let's hide it then, if someone will be interested in this feature we can always bring it back.